docs/source/create/entities/web-entity.rst

```diff
@@ -0,0 +1,130 @@
++------------------------+------------------------------------------------------------------+--------------------------+
+
+--------------------------
+Reencode media with ffmpeg
+--------------------------
+
+Here are some example commands for reencoding video files using ffmpeg:
+
++------------------------+------------------------------------------------------------------+--------------------------+
+| Encode                 | Command                                                          | Comments     |
++========================+==================================================================+==========================+
+| VP9, Opus, WebM        | ffmpeg -i "INPUTFILE" -c:v libvpx-vp9 -b:v 0 -crf 5 -vf scale=-2:600 -cpu-used 5 -row-mt 1 -c:a libopus -b:a 96K "OUTPUTFILE.webm" | "-vf scale=-2:600" scales the video down to 600p vertical resolution while keeping the aspect ratio. "-crf 5" is the video quality from 0 to 63, lower being better. For the constant quality to work the bitrate has to be set to "0" via "-b:v 0"    |
++------------------------+------------------------------------------------------------------+--------------------------+
+| VP9 (Hardware accelerated), Opus, WebM | ffmpeg -i "INPUTFILE" -c:v vp9_vaapi -b:v 2000k -c:a libopus -b:a 96K "OUTPUTFILE.webm" | VP9 hardware acceleration is currently only supported by Intel Kaby Lake or newer APUs. Hardware accelerated VP9 does not have a constant quality setting, so bitrate needs to be used instead.      |
++------------------------+------------------------------------------------------------------+--------------------------+
+| VP8 (Hardware accelerated), Opus, WebM | ffmpeg -i "INPUTFILE" -c:v vp8_vaapi -b:v 2000k -vf scale=-2:600 -c:a libopus -b:a 96K "OUTPUTFILE.webm" | VP8 hardware acceleration is currently only supported by Intel Cherryview/Braswell and newer APUs. |
++------------------------+------------------------------------------------------------------+--------------------------+
+| Theora, Opus, Matroska | ffmpeg -i "INPUTFILE" -c:v libtheora -q:v 10 -vf scale=-2:600 -c:a libopus -b:a 96K "OUTPUTFILE.mkv" | "-q:v 10" is the quality from 0 to 10, higher being better.                 |
++------------------------+------------------------------------------------------------------+--------------------------+
+
+You can use the ``ffmpeg -codecs`` command to see supported codecs.
+
+Additional info can be found with this command ``ffmpeg -h``
+    
+Alternatively you can explore these links:
+
+* https://trac.ffmpeg.org/wiki
+* https://ffmpeg.org
+* https://wiki.archlinux.org/index.php/Hardware_video_acceleration#VA-API_drivers
+
+------------------------
+Batch processing
+------------------------
+
+Here is an example shell script to batch reencode the contents of a folder on Linux using ffmpeg:
+
+You can name it ``vircadia_reencode.sh``
+
+With this code in it::
```

Comment:
```suggestion
With this code in it:
```

Reply:
This is supposed to be that way. It is a minimalized way to start a literal block. 
I will change it to the partially minimized form anyways, to make it more obvious.
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks